assets/airports/eddh.json

```diff
@@ -72,74 +94,99 @@
     "EKERN": {
       "icao": "EKERN",
       "name": "Ekern",
-      "suffix": {"05":"", "23":"", "15":"", "33":""},
+      "suffix": {"05":"8C", "23":"8B", "15":"9D", "33":"8G"},
       "rwy": {
-        "05": ["DH002", "DH004", "EKERN"],
-        "23": ["DH002", "DH004", "EKERN"],
-        "15": ["DH002", "DH004", "EKERN"],
-        "33": ["DH002", "DH004", "EKERN"]
+        "05": ["HOS", "EKERN"],
+        "23": ["DH111", "GT", "DH103", "EKERN"],
+        "15": [["DH200", "A1500+"], "HAM", "EKERN"],
```

Comment:
Fix altitude restrictions are interpreted as flight levels (hundreds of feet). Thus, since I assume you mean 1,500 feet or higher by `"A1500+"`, it should be changed to being expressed in hundreds of feet, as such: `"A15+"`.